adafruit_seesaw/tftshield18.py

```diff
@@ -66,7 +66,12 @@ class TFTShield18(Seesaw):
         # TypeError: unsupported operand type(s) for <<: 'int' and '_MockObject'
         _button_mask = 0xFF
 
-    def __init__(self, i2c_bus=board.I2C(), addr=0x2E):
+    def __init__(self, i2c_bus=None, addr=0x2E):
+        if i2c_bus == None:
+            if hasattr(board, "I2C"):
+                i2c_bus = board.I2C
```

Comment:
Oh, based on a closer look I have an additional concern.  I'd instead write
```python
if i2c_bus is None:
    try:
        i2c_bus = board.I2C()
    except AttributeError:
        raise ValueError("Board has no default I2C bus")
```
Your suggested code omits the `()` to *call* the function `board.I2C`.  Since without it (and without i2c_bus specified manually) the code cannot work at all, it is appropriate to raise an exception.

It is a general practice in CircuitPython libraries to try to catch a general and low-level exception and raise an exception which better explains the situation.

Reply:
I like that. Added in a new commit.